praw/models/reddit/modmail.py

```diff
@@ -75,6 +75,24 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
     def _info_path(self):
         return API_PATH['modmail_conversation'].format(id=self.id)
 
+    def reply(self, body, author_hidden=False, internal=False):
+        """Reply to the conversation.
+
+        :param body: The markdown formatted content for a message.
+        :param author_hidden: When True, author is hidden from non-moderators.
+        :param internal: When True, message is hidden from non-moderators.
```

Comment:
I was just looking up what these params mean. `internal` interfaces with "private moderator note". Perhaps we can use 'private_note' instead?

Also, while the API documentation doesn't say anything what happens if both boolean values are true? If they are meant to be mutually exclusive, perhaps we can have a single option, or at least a ClientException if an invalid combination are provided.